calculate.py

```diff
@@ -0,0 +1,3 @@
+
+def add(a, b):
+	return a - b
```

Comment:
Shouldn't this be a "+"? 
```suggestion
	return a + b
```